client/components/mma/upgrade/UpgradeSupportThankYou.tsx

```diff
@@ -1,40 +1,59 @@
+import { css } from '@emotion/react';
 import {
-	Button,
 	LinkButton,
 	Stack,
 	SvgCalendar,
 	SvgEnvelope,
 } from '@guardian/source-react-components';
 import { useContext } from 'react';
-import { useNavigate } from 'react-router';
+import { useLocation } from 'react-router';
 import {
 	DATE_FNS_LONG_OUTPUT_FORMAT,
 	parseDate,
 } from '../../../../shared/dates';
 import {
 	buttonCentredCss,
-	stackedButtonLeftLayoutCss,
+	stackedButtonLayoutCss,
 } from '../../../styles/ButtonStyles';
+import { iconListCss, whatHappensNextCss } from '../../../styles/GenericStyles';
 import {
-	headingCss,
-	iconListCss,
-	sectionSpacing,
-	whatHappensNextCss,
-} from '../../../styles/GenericStyles';
+	signInContentContainerCss,
+	signInCss,
+	signInHeadingCss,
+	signInParaCss,
+} from '../../shared/SignIn';
 import { Heading } from '../shared/Heading';
-import type { UpgradeSupportInterface } from './UpgradeSupportContainer';
+import { SwitchSignInImage } from '../switch/complete/SwitchSignInImage';
+import type {
+	UpgradeRouterState,
+	UpgradeSupportInterface,
+} from './UpgradeSupportContainer';
 import { UpgradeSupportContext } from './UpgradeSupportContainer';
+import {
+	headingCSS,
+	iconTextCss,
+	linkCss,
+	paragraphCss,
+	sectionSpacing,
+	withMarginParagraphCss,
+} from './UpgradeSupportStyles';
 
 export const UpgradeSupportThankYou = () => {
 	const upgradeSupportContext = useContext(
 		UpgradeSupportContext,
 	) as UpgradeSupportInterface;
 
-	const navigate = useNavigate();
-
-	const userEmailAddress = upgradeSupportContext.user;
+	const location = useLocation();
+	const routerState = location.state as UpgradeRouterState;
+	const amountPayableToday = routerState?.amountPayableToday.toFixed(2);
```

Comment:
this probably needs to be done for the whole journey, but could we do the formatting that only shows the decimal places if they are there? eg. £6 instead of £6.00

Reply:
I agree. I'll pop it on our shared document but keep this here